pyproject.toml

```diff
@@ -3,15 +3,79 @@ requires = [
     "setuptools",
     "packaging",
     "wheel",
-    "cython>=0.29.20; python_version>='3.10'",
-    "cython>=0.29.20,<3.0.0; python_version<='3.9'",
-    # See https://numpy.org/doc/stable/user/depending_on_numpy.html for
-    # the recommended way to build against numpy's C API:
+    "cython>=0.29.20",
     "numpy>=2.0.0",
     "scipy>=1.9",
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "qutip"
+dynamic = ["version"]
+description = "QuTiP: The Quantum Toolbox in Python"
+readme = "README.md"
+requires-python = ">=3.10"
+license = {file = "LICENSE.txt"}
+keywords = ["quantum", "physics", "dynamics"]
+authors = [
+  {name = "Alexander Pitchford"},
+  {name = "Paul D. Nation"},
+  {name = "Robert J. Johansson"},
+  {name = "Chris Granade"},
+  {name = "Arne Grimsmo"},
+  {name = "Nathan Shammah"},
+  {name = "Shahnawaz Ahmed"},
+  {name = "Neill Lambert"},
+  {name = "Eric Giguère"},
+  {name = "Boxi Li"},
+  {name = "Jake Lishman"},
+  {name = "Simon Cross"},
+  {email = "qutip-admin@googlegroups.com"},
+]
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Science/Research",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering",
+    "Operating System :: MacOS",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+    "Operating System :: Microsoft :: Windows",
```

Comment:
We should keep the license classifier `License :: OSI Approved :: BSD License`.

Reply:
I got a warning that it's deprecated when building when it's included.

Reply:
Ah, I see the recommendation is to specify an SPDX license directly in the license field. Let me resolve this thread.